tests/data/baseline_parameters.csv

```diff
@@ -1,2 +1,2 @@
tive_susceptibility_0_9,relative_susceptibility_10_19,relative_susceptibility_20_29,relative_susceptibility_30_39,relative_susceptibility_40_49,relative_susceptibility_50_59,relative_susceptibility_60_69,relative_susceptibility_70_79,relative_susceptibility_80,relative_transmission_household,relative_transmission_occupation,relative_transmission_random,hospitalised_fraction_0_9,hospitalised_fraction_10_19,hospitalised_fraction_20_29,hospitalised_fraction_30_39,hospitalised_fraction_40_49,hospitalised_fraction_50_59,hospitalised_fraction_60_69,hospitalised_fraction_70_79,hospitalised_fraction_80,critical_fraction_0_9,critical_fraction_10_19,critical_fraction_20_29,critical_fraction_30_39,critical_fraction_40_49,critical_fraction_50_59,critical_fraction_60_69,critical_fraction_70_79,critical_fraction_80,fatality_fraction_0_9,fatality_fraction_10_19,fatality_fraction_20_29,fatality_fraction_30_39,fatality_fraction_40_49,fatality_fraction_50_59,fatality_fraction_60_69,fatality_fraction_70_79,fatality_fraction_80,mean_time_hospitalised_recovery,sd_time_hospitalised_recovery,mean_time_critical_survive,sd_time_critical_survive,location_death_icu_0_9,location_death_icu_10_19,location_death_icu_20_29,location_death_icu_30_39,location_death_icu_40_49,location_death_icu_50_59,location_death_icu_60_69,location_death_icu_70_79,location_death_icu_80,quarantine_length_self,quarantine_length_traced_symptoms,quarantine_length_traced_positive,quarantine_length_positive,quarantine_dropout_self,quarantine_dropout_traced_symptoms,quarantine_dropout_traced_positive,quarantine_dropout_positive,quarantine_compliance_traced_symptoms,quarantine_compliance_traced_positive,test_on_symptoms,test_on_traced,trace_on_symptoms,trace_on_positive,retrace_on_positive,quarantine_on_traced,traceable_interaction_fraction,tracing_network_depth,allow_clinical_diagnosis,quarantine_household_on_positive,quarantine_household_on_symptoms,quarantine_household_on_traced_positive,quarantine_household_on_traced_symptoms,quarantine_household_contacts_on_positive,quarantine_household_contacts_on_symptoms,quarantined_daily_interactions,quarantine_days,quarantine_smart_release_day,hospitalised_daily_interactions,test_insensitive_period,test_order_wait,test_result_wait,self_quarantine_fraction,app_users_fraction_0_9,app_users_fraction_10_19,app_users_fraction_20_29,app_users_fraction_30_39,app_users_fraction_40_49,app_users_fraction_50_59,app_users_fraction_60_69,app_users_fraction_70_79,app_users_fraction_80,app_turn_on_time,lockdown_occupation_multiplier_primary_network,lockdown_occupation_multiplier_secondary_network,lockdown_occupation_multiplier_working_network,lockdown_occupation_multiplier_retired_network,lockdown_occupation_multiplier_elderly_network,lockdown_random_network_multiplier,lockdown_house_interaction_multiplier,lockdown_time_on,lockdown_time_off,lockdown_elderly_time_on,lockdown_elderly_time_off,testing_symptoms_time_on,testing_symptoms_time_off,intervention_start_time,hospital_on
```

Comment:
Set hospital to be off by default

Reply:
Should we then explicitly set all the test to have hospital_on ?

Reply:
Yes - we do this for our hospital tests anyway.

Reply:
**RESOLVED**